### ISSUE TITLE
Subcomponents don't see bindings from parent in multibindings

### ISSUE THREAD

**Issue by quiro91** [JakeWharton/dagger-reflect#208]
Parent's bindings are not present in the child subcomponent when using multibindings. This is clear in the test `ModuleWithSubcomponentsTest`, currently excluded. In order to fix it, we could check for existing binding from the parent scope in the `build()` method of `Scope.Builder`. I can raise a PR if you are OK with this solution

**Comment by JakeWharton** [comment]
Yeah I believe this is causing https://github.com/JakeWharton/dagger-reflect/blob/6ab4adefb67c315e49e303c973bab557b213afa7/integration-tests/upstream/build.gradle#L76-L77 and maybe even some of the failures above that as well.
